jenkinsfiles/RHEL_Check.groovy

```diff
@@ -193,7 +194,22 @@ pipeline {
                     }
                 }
 
+                stage('Check_code_changes') {
+                    steps {
+                        script {
+                            dir("./src") {
+                                code_changed = sh(
+                                    script: "if [ -n \"\$(git diff --name-only main | grep -v ^documentation)\" ]; then exit 1; else exit 0; fi",
```

Comment:
Do you think it requires explicit `if` and `exit`?  As far as I know bash script exits with exit code of last command.

Reply:
I provided examples here: https://github.com/oneapi-src/oneDPL/pull/235#discussion_r628030060

Reply:
Updated